### PR TITLE
Do 441 growth ci cd for bigdipper from building stage to k8s

### PR DESCRIPTION
**Pull request by praveenperera**
Github action to build docker image for bd-juno. Need to add AWS credentials as a secret.

### CODE REVIEW

**Review by luis02lopez — APPROVED** [review]

**Comment by praveenperera** [comment]
@nirtayeb I need an approval with someone with write access to merge this

**Review by nirtayeb — APPROVED** [review]

**Pull request merged**
